trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    name = @nameable.correct_name
+    name.slice(0, 10) if name.length > 10
+  end
```

Comment:
- Optional
To keep your code clean, you can also use the ternary operator. Like this ⬇️
`@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`